README.rst

```diff
@@ -29,64 +31,138 @@ Contributing
+``setup.py``), you must be using SCons 4.0 or newer.
+If you are using an older version of SCons,
+see the `Manual Installation`_ instructions.
+
+For installing via ``pip``, you need to point directly to
+the code repostory, as this collection is not available on
+the `Python Package Index <https://pypi.org>`_::
+
+    python -m pip install --user git+https://github.com/SCons/scons-contrib.git
+
+This puts all of the tools into the ```Tool``` subdirectory of
+the SCons installation, in other words, in the same place
+SCons looks for its own built-in tools.  Usually you don't have
+permission to write here, so the ``--user`` option is used
+to put them into a parallel writable location specific to your account.
+
+The SCons project generally recommends that you use a Python
+virtualenv to set up SCons, because you can keep the dependencies
+isolated from others. Since many of the contributed tools
+use other facilities (for example the Sphinx tools require
+Sphinx itself, which requires quite a few other Python packages)
+this avoids installing a lot of dependnecies
+into your system install of Python
+(even if they went into the user location).
+Inside an activated virtualenv you can use the somewhat simpler::
 
-::
+    pip install git+https://github.com/SCons/scons-contrib.git
+
+There's an alternate installation style you can perform as well -
+clone the scons-contrib repository yourself and then run
+setup manually::
 
+    git clone https://github.com/SCons/scons-contrib.git
+    cd scons-contrib
     python setup.py install
 
+The Python packaging experts no longer recommend any project
+call ``setup.py`` manually, and since the ``pip install``
+command pointing to the remote code repository does basically
+the same thing behind the covers but more cleanly/safely,
+this alternate install is no longer recommended, unless
+there's a reason ``pip`` cannot be used to point to a code
+repository (company security policy, perhaps?)
+
+
 Manual Installation
-===================
+*******************
+
+`Installing via pip`_ results in all of the contrib tools being
+installed at once. If you just want a specific tool or two,
+a manual installation may be a better choice.
+
+To do this, clone the repository::
+
+    git clone https://github.com/SCons/scons-contrib.git
+
+and copy the required tool directory into one of the locations
+SCons searches for tools. Roughly speaking, there are three
+system-specific locations that are searched by default:
+a system-wide location, a user-account location, and inside
+a given project. These default locations are in the man page at
+`<https://scons.org/doc/production/HTML/scons-man.html#opt-site-dir>`_
+
+In addition to the defaults, the project-level site directory
+can be changed by supplying the ``--site-dir`` option,
+and SCons can also be given additional locations to look for
+tools using the ``toolpath`` parameter when creating a
+Construction Environment. See the manpage for details.
```

Comment:
I'd leave this <<<<<=== text in. It's the simplest method to install a tool... and then include link to more info about the site dir? So they can get er done without having to click through to another doc..

Reply:
Sorry, leave what in?

Reply:
Pushed a revised version which calls out `site_scons/site_tool`, before pointing off to the "for more info" stuff.